packages/atomic/cypress/e2e/pager.cypress.ts

```diff
@@ -127,4 +127,45 @@ describe('Pager Test Suites', () => {
       CommonAssertions.assertContainsComponentError(PagerSelectors, true);
     });
   });
+
+  describe('Options for previous next button icons', () => {
+    before(() => {
+      new TestFixture().with(addPager()).init();
+    });
+
+    it('should expose shadow part for next icon', () => {
+      cy.get('atomic-pager').shadow().find('[part="next-button-icon"]').click();
+      PagerAssertions.assertRenderPager(2);
+    });
+
+    it('should expose shadow part for previous icon', () => {
+      cy.get('atomic-pager')
+        .shadow()
+        .find('[part="previous-button-icon"]')
+        .click();
+      PagerAssertions.assertRenderPager(1);
```

Comment:
This will also create an `it` block in the current `it` block.

Reply:
Ah, notice now that assertRenderPager has an it - but it doesn't render in the cypress ui. 
Besides assertPagerSelected requires it - errors inside describe.